packages/babel-standalone/scripts/pluginConfig.json

```diff
@@ -0,0 +1,87 @@
+{
+  "all": [
```

Comment:
Can't this file just be the array?

Reply:
Oh yes. I first had more keys in there, as I wanted to also specify which plugins go to which preset, but that does not work because of plugin options and other business logic.
Will revert.

Reply:
Done